helm/values.yaml

```
@@ -992,6 +992,7 @@ zenml:
   service:
     type: ClusterIP
     port: 80
+    annotations:
```

```suggestion
    annotations: {}
```

I'm worried that a null default value might result in an error if this is not set.

---

Thanks @stefannica I have added your suggestion!